operating/manage-vulnerabilities/scan-rhcos-node-host.adoc

```diff
@@ -7,9 +7,11 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-For {ocp}, {op-system-first} is the only supported operating system for control plane. Whereas, for node hosts, {ocp} supports both {op-system} and {op-system-base-full}.
+For {ocp}, {op-system-first} is the only supported operating system for control plane. For node hosts, {ocp} supports both {op-system} and {op-system-base-full}.
 With {rh-rhacs-first}, you can scan {op-system} nodes for vulnerabilities and detect potential security threats.
 
+//Any changes needed for 4.6 Scanner V4 for RHCOS nodes?
```

Comment:
From a user perspective, no. Functionally, not much has changed. It might be interesting to add that Node Indexing with Scanner V4 is using VEX feeds now (ref https://github.com/openshift/openshift-docs/pull/83406).

Reply:
OK, so is this statement no longer accurate? 

> Then it matches vulnerabilities for identified components by using both RHEL and OpenShift 4.X Open Vulnerability and Assessment Language (OVAL) v2 security data streams. 

In that other PR, @RTann said to keep the OVAL v2 language. I might not be understanding some nuance here though.

Reply:
That's interesting. I double checked the discussions in the other thread, thanks for flagging that.
Let me explain a bit:
Technically, RHCOS Node Indexing uses whatever Scanner v4 exposes, be it VEX or OVAL v2.
Scanner v4 currently doesn't distinguish between an Image and Node Scan (this will likely change in 4.7, though).

From my understanding, we're deprecating OVAL v2 in favor of VEX in Scanner v4, so I would've written it like you did. Maybe Ross can shed some light on this :smile:

Reply:
We still use OVAL data for StackRox Scanner. I believe I interpreted the update to that other PR to be referring to StackRox Scanner's RHCOS support, and not Scanner V4's